CHANGELOG.md

```diff
@@ -12,6 +12,18 @@ This project tries to follow [SemVer 2.0.0](https://semver.org/).
 	https://changelog.md/
 -->
 
+## v1.6.0 (WIP)
+
+- Regenerated rest clients for: (#91)
+
+  - `wharf-api` v5.0.0
+  - `wharf-provider-github` v2.0.0
+  - `wharf-provider-gitlab` v1.3.0
+  - `wharf-provider-azuredevops` v2.0.1
+
+- Changed function calls and object references to match new names in the
+  generated code. (#91)
```

Comment:
I think this note should be merged with the above note. Gives it more context to why it was changed.

nit: "Changed object references" is a bit vague IMO. Suggest to instead phrase it like "Changed function and type names to use the new names from the generated wharf-api v5.0.0 code."